Taskfile.yml

```diff
@@ -0,0 +1,144 @@
+version: "3"
+
+vars:
+  # Variables for Docker image
+  REGISTRY: ghcr.io/openmcp-project
+  IMAGE_NAME: mcp-ui-frontend
+  IMAGE_TAG: '{{.TAG | default "latest"}}'
+  DOCKER_IMAGE: "{{.REGISTRY}}/{{.IMAGE_NAME}}:{{.IMAGE_TAG}}"
+
+  # Variables for local testing
+  LOCAL_REGISTRY: localhost:5000
+  LOCAL_IMAGE_NAME: mcp-ui-frontend
+  LOCAL_DOCKER_IMAGE: "{{.LOCAL_REGISTRY}}/{{.LOCAL_IMAGE_NAME}}:{{.IMAGE_TAG}}"
```

Comment:
```suggestion
```

You dont need to add a local registry & image for building the ocm component locally. You can just use the oci image from remote and build it locally with the remote image.